### ISSUE TITLE
[Bug] Zoom rectangle off position with `pyqtgraph` 0.13.1

### ISSUE THREAD

**Issue by LukePiewalker** [Ulm-IQO/qudi-iqo-modules#50]
### Version

current main

### What is affected by the bug?

When applying a zoom, the zoom rectangle is not displayed on the location of zoom, as has been encountered already some time ago.

### When does the bug occur?

This is happening after a fresh install of qudi-core and qudi-iqo-modules in an anaconda environment with python 3.8.12
The resulting pyqtgraph version as of now is 0.13.1.
Downgrading to pyqtgraph version 0.12.4 resolves the issue, so I guess it's pyqtgraph related again, like last time.

### How do we replicate the issue?

I guess, as stated above? Reinstalling qudi.

### Expected behavior

The zoom rectangle should appear at the location of the mouse.

**Comment by Neverhorst** [comment]
`pyqtgraph` is driving me nuts. Undocumented breaking changes left and right...
So I introduced a fix for this issue when we upgraded from `0.12.3` to `0.12.4` because they completely changed scene/view coordinate mapping for this rectangle drawing. This was quite the major breaking change with no mention in the changelog whatsoever. Now AFTER `0.12.4`, i.e. from `0.13.0` onward, they changed it back to how it was before... of course also without any mention in the changelog. Meaning my workaround breaks again.

TL;DR: `pyqtgraph` messed up and made specifically version `0.12.4` incompatible with all other versions regarding this scaleBox drawing. So I will make my version check workaround only check for this specific version and throw an error telling the user to update `pyqtgraph`.

**Comment by Neverhorst** [comment]
Should be fixed by PR Ulm-IQO/qudi-core#39

**Comment by LukePiewalker** [comment]
Agree! Is fixed according to my tests. Thanks a lot @Neverhorst !!

**Comment by Neverhorst** [comment]
You are welcome. Maybe, let's merge the PR first before closing this issue. (Should close automatically if PR is merged if I recall GitHub documentation correctly).
